megatron/core/transformer/custom_layers/transformer_engine.py

```diff
@@ -894,7 +894,7 @@ def te_checkpoint(
     def get_cpu_offload_context(
         enabled, num_layers, model_layers, activation_offloading, weight_offloading
     ):
-        if _te_version > packaging.version.Version("1.8.0"):
+        if _te_version > packaging.version.Version("1.9.0"):
```

Comment:
"> 1.9.0" is not sufficient. In many cases we have TE versions like `v1.9.0+commit_id`, which is greater than 1.9.0.
I suggest it to be ">= 1.10.0.dev0"